ibis/backends/flink/registry.py

```diff
@@ -96,6 +132,29 @@ def _timestamp_from_unix(translator: ExprTranslator, op: ops.Node) -> str:
     return f"TO_TIMESTAMP_LTZ({numeric}, {precision})"
 
 
+def _timestamp_from_ymdhms(
+    translator: ExprTranslator, op: ops.temporal.TimestampFromYMDHMS
+) -> str:
+    date_string = (
+        "CONCAT("
+        + ", '-', ".join(
+            [
+                f"CAST({translator.translate(e)} AS STRING)"
+                for e in [op.year, op.month, op.day]
+            ]
+        )
+        + ", ' ', "
+        + ", ':', ".join(
+            [
+                f"CAST({translator.translate(e)} AS STRING)"
+                for e in [op.hours, op.minutes, op.seconds]
+            ]
+        )
+        + ")"
+    )
```

Comment:
All the punctuation here is a bit challenging to read :)

How about this?

```suggestion
        year, month, day = [
            f"CAST({translator.translate(e)} AS STRING)"
            for e in [op.year, op.month, op.day]
        ]
        hours, minutes, seconds = [
            f"CAST({translator.translate(e)} AS STRING)"
            for e in [op.hours, op.minutes, op.seconds]
        ]
        date_string = f"CONCAT({year}, '-', {month}, '-', {day}, ' ', {hours}, ':', {minutes}, ':', {seconds})"
```

Reply:
Thanks! I applied your suggestion.